src/frontend/src/components/FillForms/index.tsx

```diff
@@ -24,6 +24,19 @@ export default class FillForms extends React.Component<Props, State> {
                 files for all cases that have charges eligible for expungement.
               </p>
 
+              <p className="mb3">
+                It includes cases that have an outstanding balance. These
+                balances must be paid off before filing the paperwork. Consult
+                the section on{" "}
+                <Link
+                  className="link bb hover-dark-blue"
+                  to="/manual#paybalances"
+                >
+                  paying balances
+                </Link>{" "}
+                in the manual.
```

Comment:
```suggestion
                in the Manual.
```

Reply:
It's capitalized in other spots on the page and I think that's probably a good pattern to stick to.